### PR TITLE
Automatically update charts in 2.1 when they change in 1.0, 1.1 or 2.0

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by dgn — APPROVED** [review]
thanks a lot! this is quite annoying

File: prow/config/postsubmits.yaml [hunk 1]
```diff
@@ -417,6 +417,53 @@ postsubmits:
...
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.1"
+        imagePullPolicy: Always
+        command:
+        - ../test-infra/tools/automator.sh
+        - -o maistra
+        - -r istio-operator
+        - -f /creds-github/github-token
+        - -c make clean update-1.0-charts update-1.1-charts update-2.0-charts
```
Comment (jwendell) [inline prow/config/postsubmits.yaml:447]: This is not going to work as intended. Let's say 1.1 branch was updated. Then this job will run on the 1.1 branch, and these targets do not exist. You will want to run this job on the 2.1 branch, right?

Comment (jwendell) [reply prow/config/postsubmits.yaml:447]: You probably want to add `-b maistra-2.1` to this script.

Comment (jwendell) [reply prow/config/postsubmits.yaml:448]: Also, add `-l auto-merge`.
And see this comment: https://github.com/maistra/istio-operator/pull/746#issuecomment-862489962

This is part of MAISTRA-2445.

Comment (luksa) [reply prow/config/postsubmits.yaml:448]: Thanks. I missed the fact that the target branch isn't defined anywhere.

**Comment by luksa** [comment]
@jwendell I waited for you to review this, since this was the first time I touched this part of the code. If you think this is now ok, should I add the same kind of jobs for the other branch combinations (e.g. update maistra-2.0 branch when 1.1 or 1.0 branches change, and so on)?

**Comment by jwendell** [comment]
@luksa It LGTM. Just add the `-l auto-merge` argument. If you want to test this in practice, you can push something to the `playground` branch and observe this post-submit job. It should 1) run and 2) create a proper PR in 2.1 branch.

**Review by jwendell — APPROVED** [review]

**Pull request merged**